packages/tocco-ui/src/Panel/Panel.js

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+/**
+ * <Panel/> is used to conceal and display related content alternating by interaction.
+ */
+class Panel extends React.Component {
+  state = {
+    isOpen: this.props.isOpen
+  }
+
+  toggleOpenState = () => {
```

Comment:
toggleOpenState = () => {
   if (this.props.isToggleable) {
    this.setState(prevState => ({
      isOpen: !prevState.isOpen
...

Reply:
I will use conditional statements over short circuit to respect ESLint.

Reply:
see 88e5dbe45b2124693323e37e12e97f459bcf44ec